RussianFilter/sections/general_extensions.txt

```diff
@@ -887,9 +887,9 @@ fclmnews.ru#$#.col_02 > div.top_3 { position: absolute!important; left: -3000px!
 ! https://forum.adguard.com/index.php?threads/10880/
 sinoptik.ua#$#iframe[bb] { position: absolute!important; left: -3000px!important; }
 sinoptik.com.ru#$#iframe[bb] { visibility: hidden!important; }
-! https://github.com/AdguardTeam/AdguardFilters/issues/4368
-arendator.ru#$#.arcticmodal-container { display: none!important; }
-arendator.ru#$#.arcticmodal-overlay { display: none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/4368 https://github.com/AdguardTeam/AdguardFilters/issues/31764
+gopokemongo.ru,arendator.ru#$#.arcticmodal-container { display: none!important; }
+gopokemongo.ru,arendator.ru#$#.arcticmodal-overlay { display: none!important; }
 arendator.ru#$#body { overflow: visible!important; }
```

Comment:
Please add domain also to this element or else scrolling is broken.

Reply:
I did not observe this in myself. Also checked with/without `gopokemongo.ru#$#body { overflow: visible!important; }` before commit, but i add this now. Thanks!